composer.json

```
@@ -15,10 +15,10 @@
     "require": {
         "php": "^7.2||^8.0",
         "friendsofphp/php-cs-fixer": "^2.1||^3.0",
-        "symfony/event-dispatcher": "^4.4||^5.0||^6.0",
-        "symfony/finder": "^4.4||^5.0||^6.0",
-        "symfony/console": "^4.4||^5.0||^6.0",
-        "symfony/filesystem": "^4.4||^5.0||^6.0"
+        "symfony/event-dispatcher": "^4.4||^5.0||^6.0|^7.0",
+        "symfony/finder": "^4.4||^5.0||^6.0|^7.0",
+        "symfony/console": "^4.4||^5.0||^6.0|^7.0",
+        "symfony/filesystem": "^4.4||^5.0||^6.0|^7.0"
```

```suggestion
        "symfony/event-dispatcher": "^4.4||^5.0||^6.0||^7.0",
        "symfony/finder": "^4.4||^5.0||^6.0||^7.0",
        "symfony/console": "^4.4||^5.0||^6.0||^7.0",
        "symfony/filesystem": "^4.4||^5.0||^6.0||^7.0"
```

`|` should work according to the [doc](https://getcomposer.org/doc/articles/versions.md#version-range) for BC reason, but `||` is the official way to do a logical OR